### PR TITLE
Delete Reflect/Object comparison page, align all Reflect pages

### LINKED ISSUE #21652: "Comparing Reflect and Object methods" page has overflowing table
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Reflect/Comparing_Reflect_and_Object_methods

### What specific section or headline is this issue about?

The main section, specifically the big comparison table.

### What information was incorrect, unhelpful, or incomplete?

The right-most column in the table is cut off due to the width of the table wrapper. Have a look:

image

The width of the table wrapper with selector `div.table-scroll` is set to `width: 100%;`.

### What did you expect to see?

I expect to see the following. Here, the table wrapper with selector `div.table-scroll` has the width set with `width: min-content;`

image

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

I'm using Firefox, 1080p display, window maximized (tiling window manager).

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/javascript/reference/global_objects/reflect/comparing_reflect_and_object_methods`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Reflect/Comparing_Reflect_and_Object_methods
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/global_objects/reflect/comparing_reflect_and_object_methods/index.md
* Last commit: https://github.com/mdn/content/commit/968e6f1f3b6f977a09e116a0ac552459b741eac3
* Document last modified: 2022-09-09T05:25:05.000Z

</details>

### PR DESCRIPTION
**Pull request by Josh-Cena**
### Description

Another wave of improving MDN meta-programming docs (see https://github.com/mdn/content/pull/22085). Fix https://github.com/mdn/content/issues/21652

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/reflect/comparing_reflect_and_object_methods/index.md [hunk 1]
```diff
@@ -1,203 +0,0 @@
----
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/reflect/comparing_reflect_and_object_methods/index.md:1]: This looks weird. It looks like the page got emptied but not deleted or redirected. (Did you use `yarn content delete <...> --redirect <...>`?)

File: files/en-us/_wikihistory.json [hunk 2]
```diff
@@ -30518,6 +30481,43 @@
       "asadotzler"
     ]
   },
+  "Web/API/Document_object_model/Using_the_Document_Object_Model": {
```
Comment (teoli2003) [inline files/en-us/_wikihistory.json:30484]: Is something applied (husky?) on this file that wasn't before? This is not a big deal (as it only sorts capitalized letters differently, and this appears when there is case incoherence. But that's the 2nd PR today that shows this behaviour.

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/javascript/reference/global_objects/reflect/comparing_reflect_and_object_methods/index.md (hunk 1 above)
Comment (Josh-Cena) [reply files/en-us/web/javascript/reference/global_objects/reflect/comparing_reflect_and_object_methods/index.md:1]: This file surely looks deleted and redirected.

image

File: files/en-us/_wikihistory.json (hunk 2 above)
Comment (Josh-Cena) [reply files/en-us/_wikihistory.json:30484]: No—I just ran `yarn content delete ... --redirect ...` and here it is.

**Review by teoli2003 — APPROVED** [review]
Yes, it was deleted. (My bad)

**Pull request merged**